climate.py

```diff
@@ -179,15 +203,25 @@ def update(self):
         This is the only method that should fetch new data for Home Assistant.
         """
         state = self._nobo.get_current_zone_mode(self._id, dt_util.as_local(dt_util.now()))
+        self._current_mode = HVAC_MODE_AUTO
```

Comment:
```suggestion
        self._current_mode = HVAC_MODE_AUTO
        self._current_operation = PRESET_NONE
```
Suggest setting initial operation as well